circle.yml

```diff
@@ -22,6 +23,11 @@ test:
   # Grep error on the documentation
   override:
     - cat ~/log.txt && if grep -q "Traceback (most recent call last):" ~/log.txt; then false; else true; fi
+deployment:
```

Comment:
will this only be run in the override returns true? Then we need to fix #5953 before the docs get uploaded :-/